src/farmers/farmerbot/farmerbot_quick.md

```diff
@@ -40,76 +64,212 @@ Here are some examples to guide you:
 
+Once you've verified that the Farmerbot runs properly, you can stop the Farmerbot and go to the next section to set a Farmerbot service. This step will ensure the Farmerbot keeps running after exiting the VM.
 
+### Set a systemd Service
 
-## Create the Configuration Files
+It is highly recommended to set a Ubuntu systemd service to keep the Farmerbot running after exiting the VM.
 
-* Create the parent folder
+* Create the service file
   * ```
-    mkdir -p farmerbot_docker && cd $_
+    nano /etc/systemd/system/farmerbot.service
     ```
-    * Note: you can choose a different name for your parent folder
-* Pull the file creator image from the GitHub Container Registry
-  * ```
-    docker pull ghcr.io/threefoldtech/farmerbot_config:0.2.0
+* Set the Farmerbot systemd service
+    
     ```
-* Run the file creator image and answer the questions to generate the files
-  * ```
-    docker run --name fbot_config_container -v ./:/farmerbot -ti ghcr.io/threefoldtech/farmerbot_config:0.2.0
+    [Unit]
+    Description=ThreeFold Farmerbot
+    StartLimitIntervalSec=0
+
+    [Service]
+    Restart=always
+    RestartSec=5
+    StandardOutput=append:/root/farmerbotfiles/farmerbot.log
+    StandardError=append:/root/farmerbotfiles/farmerbot.log
+    ExecStart=/usr/local/bin/farmerbot run -e /root/farmerbotfiles/.env -c /root/farmerbotfiles/config.yml -d
+
+    [Install]
+    WantedBy=multi-user.target     
+    ```
+* Enable the Farmerbot service
+    ```
+    systemctl daemon-reload
+    systemctl enable farmerbot
+    systemctl start farmerbot
+    ```
+* Verify that the Farmerbot service is properly running
+    ```
+    systemctl status farmerbot
     ```
-    * The `config/config.md` and `.env` files are saved in the parent folder
 
-> Note: It is can be a good idea to set only one node with `yes` for `never_shutdown`. A node with low power consumption is obviously a wise choice.
+### Check the Farmerbot Logs
 
+Once you've set a Farmerbot systemd service [as show above](#set-a-systemd-service), the Farmerbot will start writing logs to the file `farmerbot.log` in the directory `farmerbotfiles`.
 
+Thus, you can get more details on the operation of the Farmerbot by inspecting the log file. This can also be used to see the **Farmerbot Report Table** as this table is printed in the Farmerbot log.
 
-## Run the Farmerbot
+* See all logs so far
+  ```
+  cat ~/farmerbotfiles/farmerbot.log
+  ```
+* See the last ten lines and new logs as they are generated
+  ```
+  tail -f ~/farmerbotfiles/farmerbot.log
+  ```
+* See all logs and new lines as they are generated
+  ```
+  tail -f -n +1 ~/farmerbotfiles/farmerbot.log
+  ```
+* See the last report table
+  ```
+  tac farmerbot.log | grep -B5000 -m1 "Nodes report" | tac
```

Comment:
update path to the logs to match the previous one by adding `~/farmerbotfiles/`.

`tac ~/farmerbotfiles/farmerbot.log | grep -B5000 -m1 "Nodes report" | tac`

Reply:
Good idea. Done.